serverless/src/exportSearchRequest/handler.js

```diff
@@ -0,0 +1,118 @@
+import AWS from 'aws-sdk'
+import { v4 as uuidv4 } from 'uuid'
+
+import { determineEarthdataEnvironment } from '../util/determineEarthdataEnvironment'
+import { getApplicationConfig } from '../../../sharedUtils/config'
+import { getDbConnection } from '../util/database/getDbConnection'
+import { getJwtToken } from '../util/getJwtToken'
+import { getVerifiedJwtToken } from '../util/getVerifiedJwtToken'
+import { getSqsConfig } from '../util/aws/getSqsConfig'
+import getSearchExportQueueUrl from '../util/aws/getSearchExportQueueUrl'
+import { parseError } from '../../../sharedUtils/parseError'
+
+// adapter for Amazon Simple Queue Service (SQS)
+let sqs
+
+/**
+ * @description Request an export from cmr-graphql
```

Comment:
I don't think we use this description tag anywhere else. Whether or not thats right, we should be consistent. If we want to use that flag in earthdata search, we should do it everywhere.

Reply:
done with https://github.com/nasa/earthdata-search/pull/1630/commits/d0f1eef4cadd01beb23d8ad674f62896c654d2dd